airbyte-scheduler/src/main/java/io/airbyte/scheduler/persistence/JobPersistence.java

```diff
@@ -143,6 +143,6 @@
    */
   Optional<State> getCurrentState(UUID connectionId) throws IOException;
 
-  Optional<Job> getOldestPendingJob() throws IOException;
+  Optional<Job> getNextJob() throws IOException;
```

Comment:
changing name to so that you have to go read what the rules of the method.

Reply:
yep good name.